src/lib/command-framework/apify-command.ts

```diff
@@ -380,13 +390,24 @@ export abstract class ApifyCommand<T extends typeof BuiltApifyCommand = typeof B
 		let baseDefinition = `${nameOverride || this.ctor.name}`;
 
 		if (this.ctor.args) {
+			let setCatchAll = false;
+
 			for (const [key, internalBuilderData] of Object.entries(this.ctor.args)) {
 				if (typeof internalBuilderData === 'string') {
 					throw new RangeError('Do not provide the string for the json arg! It is a type level assertion!');
 				}
 
 				// We mark all args as optional, even if they are not, to pretty-print a help message ourselves
-				baseDefinition += ` [${key}]`;
+				if (internalBuilderData.catchAll) {
+					if (setCatchAll) {
+						throw new RangeError('Only one catch-all argument is allowed in a command');
+					}
+
+					setCatchAll = true;
+					baseDefinition += ` [${key}...]`;
+				} else {
+					baseDefinition += ` [${key}]`;
+				}
 			}
```

Comment:
I might be missing the point here, but what is the `setCatchAll` local variable for? It seems to be noop

Reply:
It's set in the event someone makes 2 catch-all flags for whatever reason (although technically shouldn't be an issue, who knows 🤷)